src/components/AlertBox/index.tsx

```diff
@@ -23,10 +31,19 @@ export const AlertBox: FC<AlertBoxProps> = ({
   withIcon = true,
   ...restProps
 }) => {
+  const { colorMode } = useColorMode()
+
+  const isDarkMode = colorMode === "dark"
+
   if (status === "magic") {
     return (
-      <Alert status="info" {...restProps} backgroundColor="brand.100">
-        {withIcon && <AlertIcon color="brand.500" />}
+      <Alert
+        status="info"
+        {...restProps}
+        backgroundColor={isDarkMode ? "rgba(255, 255, 255, 0.08)" : "brand.100"}
+        borderColor={isDarkMode ? "brand.400" : "brand.500"}
```

Comment:
Other statues allow to change `borderColor` and `backgroundColor` of the `Alert`. For `magic` status it does not work, because we override potential `backgroundColor` and `borderColoer` that could be in `...restProps`. My propose to fix this is to simply place the `...restProps` at the end:

```suggestion
        backgroundColor={isDarkMode ? "rgba(255, 255, 255, 0.08)" : "brand.100"}
        borderColor={isDarkMode ? "brand.400" : "brand.500"}
        {...restProps}
```